admiral/pkg/clusters/handler.go

```diff
@@ -106,9 +106,27 @@ func getDestinationRule(se *v1alpha32.ServiceEntry, locality string, gtpTrafficP
 func getOutlierDetection(se *v1alpha32.ServiceEntry, locality string, gtpTrafficPolicy *model.TrafficPolicy) *v1alpha32.OutlierDetection {
 
 	outlierDetection := &v1alpha32.OutlierDetection{
-		BaseEjectionTime:      &types.Duration{Seconds: 300},
+		BaseEjectionTime:         &types.Duration{Seconds: 300},
```

Comment:
May be we can define constants for all the defaults and use them in our tests as well.